### DISCUSSION TITLE
Having a Sync problem

### DISCUSSION THREAD

**Discussion by mrweneafrica** [community/community#88646] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

I am using cpanel, github desktop and visual studio code, to create my website. I create files and folders directly on vCode, then push to github.com using github desktop. After creating folders I realized I want to change all my folders from uppercase and capitalized to lowercase. Eg (CSS to css, En to en, Media to media). I did the renaming in vCode, but unfortunately it just doesn't reflect on github.com

How can I solve this issue. Because it's affecting my URLs.

**Comment by Shahnawazk7944** [comment]
hey @mrwene
Stage the changes (Ctrl+Shift+G) -> Commit the changes (add the commit message e.g. renaming project folder and files) (Ctrl+Enter) -> Push to GitHub (Ctrl+Shift+P), and your changes will be reflected on GitHub.

**Reply by mrweneafrica** [reply]
hello, thank you. 

I am not very used to using this method I am using now. May I know if I should do this in vCode? Or GithubDesktop?

**Reply by Shahnawazk7944** [reply]
In GitHub desktop, you will see Commit button in bottom right corner just add a commit message and Commit the changes then you will see Push button pop-up click on it your changes will be reflected to GitHub.

**Reply by mrweneafrica** [reply, marked as answer]
So unfortunately that didnt work, i found a way, you just create a file inside the folder. Then head to edit the file name on github.com. Then you if the folder is Folder/File-name then you move the edit curser to just before File (in the beginning of the file name after the slash) and backspace, then you can edit the folder name. Very complicated but works. They need to make it easier to do this. Maybe directly on vCode when you edit the folder it should reflect in the commits.

**Reply by Shahnawazk7944** [reply]
Ohh great 👍 thanks for the solution.
